pages/linux/pinout.md

```diff
@@ -0,0 +1,12 @@
+# pinout
+
+> View the current Raspberry Pi's GPIO pin-out information on the terminal with an ASCII diagram
+> More information: <https://www.raspberrypi.org/documentation/computers/os.html#gpio-pinout>.
+
+- View the pinout information and GPIO header diagram for the current Raspberry Pi:
+
+`pinout`
+
+- open https://pinout.xyz/ in the default broswer:
```

Comment:
```suggestion
- Open https://pinout.xyz/ in the default broswer:
```

Reply:
I don't see the value of this command.

It doesn't really give any help, at most the pinout.xyz should be the more information link.

Reply:
But <pinout.xyz> is just a part of the command. It's basically a shortcut and I'm sure it can be very useful.
Look at `gh browse` for example. The whole purpose of this command is to open stuff in the default browser.

Reply:
I considered which one to use and thought that to provide both sources of info and let individuals have access to both

Reply:
This seems fine to me, though @blueskyson's suggestion should be applied.